### PR TITLE
[pallet-revive] set logs_bloom

### PR DESCRIPTION
**Pull request by pgherveou**
Set the logs_bloom in the transaction receipt

### CODE REVIEW

**Comment by athei** [comment]
Still has some compilation errors.

File: prdoc/pr_6460.prdoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title: '[pallet-revive] set logs_bloom'
+doc:
+- audience: Runtime Dev
+  description: 'Set the logs_bloom in the transaction receipt
+
+    '
```
Comment (xermicus) [inline prdoc/pr_6460.prdoc:6]:
```suggestion
  description: Set the logs_bloom in the transaction receipt
```

**Review by xermicus — COMMENTED** [review]
lgtm

**Review by athei — APPROVED** [review]

**Review by xermicus — APPROVED** [review]

**Pull request merged**